test/controllers/ccmenu_controller_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
+
+module Shipit
+  class CcmenuControllerTest < ActionController::TestCase
+    def get_project_from_xml(xml)
+      Hash.from_xml(xml)['Projects']['Project']
+    end
```

Comment:
style nitpick: I usually move these to the bottom of the test class, and mark them as `private`.